app/src/main/java/com/example/myspacexdemoapp/api/SpaceXApi.kt

```diff
@@ -10,10 +10,10 @@ import io.reactivex.rxjava3.core.Observable
 import java.text.SimpleDateFormat
 import java.util.Locale
 
-class SpaceXApi(private val apolloClient: ApolloClient) : ISpaceXApi {
+open class SpaceXApi(private var apolloClient: ApolloClient) : ISpaceXApi {
```

Comment:
What are the reasons to make `SpaceXApi` open and `ApolloClient` var?
If it is only to run tests just add `org.mockito:mockito-inline` lib.